include/fmt/printf.h

```diff
@@ -405,7 +405,9 @@ void basic_printf_context<OutputIt, Char>::parse_flags(format_specs& specs,
       specs.fill[0] = '0';
       break;
     case ' ':
-      specs.sign = sign::space;
+      if (specs.sign != sign::plus) {
```

Comment:
from the opengroup description: "This means that if the <space> and '+' flags both appear, the <space> flag shall be ignored."